src/components/file-dropper/file-dropper.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { FileError, FileRejection, useDropzone } from 'react-dropzone';
+import styled from 'styled-components';
+import { theme } from '../layout/theme';
+import { Illustration, IllustrationSize } from '../illustration/illustration';
+import { IllustrationType } from '../illustration/illustration-set';
+import { Trans } from 'gatsby-plugin-react-i18next';
+import { TextStyles } from '../typography/typography-v2';
+
+interface FileDropperProps {
+  /**
+   * (Optional) Pass in a valid illustration keyword to show the according illustration.
```

Comment:
`(Optional)` is not needed here, that's already clear with the type.

Reply:
I did that before. That's not clear from the table being generated later so I think it's good for the story to include.